R/rules-spaces.R

```diff
@@ -140,20 +140,34 @@ remove_space_before_opening_paren <- function(pd_flat) {
   pd_flat
 }
 
-remove_space_after_opening_paren <- function(pd_flat) {
-  paren_after <- pd_flat$token %in% c("'('", "'['", "LBB")
+remove_space_after_opening_paren <- function(pd_flat, strict) {
+  braces <- c("'('", "'['", "LBB")
+  paren_after <- pd_flat$token %in% braces
   if (!any(paren_after)) {
     return(pd_flat)
   }
+  # remove blank lines after opening braces
+  if (strict) {
```

Comment:
TBH, I don't think we should have `strict` parameter here. Such blank lines should always be removed. But I wanted to keep them in to align with other functions in this family. 

If you disagree, and we do need to support this parameter, then there need to be some additional tests to cover strict set to `FALSE`.